benchmarks/babelstream/src/babelStreamCommon.hpp

```diff
@@ -28,55 +28,105 @@ namespace
     [[maybe_unused]] constexpr auto minArrSize = 1024 * 128;
 
     // Scalar value for Mul and Triad kernel parameters.
-    [[maybe_unused]] constexpr auto scalarVal = 2.0f;
+    [[maybe_unused]] constexpr double scalarVal = 0.4;
 
     // Block thread extent for DotKernel test work division parameters.
     [[maybe_unused]] constexpr auto blockThreadExtentMain = 1024;
+    [[maybe_unused]] constexpr auto dotGridBlockExtent = 256;
```

Comment:
feel free to add a extent for CPUs too to support CPU dot execution required for the verification.

Reply:
additional test case is added for blocksize = 1 for Dot Kernel. Hence Dot kernel is tested for CPU backend and for blocksize =1.

Reply:
Ok, now all backends are used and tested. [ `AccCpuThreads` backend is very slow, but passed the pipeline. ]